ghosts can spawn as nymphs from diona players

bigfunguss opened the pull request:
:cl: Funguss
rscadd: allows ghosts to spawn as diona nymphs from a humanoid gestalt.
/:cl:

Spookerton commented:
Considering the viability of this at the moment, what with the high limits, lore madness, and katamari potential.

bigfunguss committed:
nymph ghost verb

ghosts can join as nymphs from existing humanoid gestalts, with a total server cap of five at a time

Spookerton commented:
Lore noped -- Adolescent gestalts don't generate nymphs. Closing.
